linkerd/meshtls/boring/src/client.rs

```diff
@@ -76,25 +81,36 @@ where
             let conn = connector.map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
             let config = conn
                 .configure()
-                .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
-            let io = tokio_boring::connect(config, server_name.as_str(), io)
+                .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?
+                // Switch off DNS san Verification based on the hostname
```

Comment:
tiny nit: I think SAN is also an acronym (for "Subject Alternate Name"):
```suggestion
                // Switch off DNS SAN Verification based on the hostname
```